src/azure-cli/azure/cli/command_modules/botservice/_params.py

```diff
@@ -63,7 +63,6 @@ def load_arguments(self, _):
         c.argument('password', options_list=['-p', '--password'], help='The Microsoft account (MSA) password for the bot. Used to authorize messages being sent to the bot.')
         c.argument('tags', arg_type=tags_type)
         c.argument('language', options_list=['--lang'], arg_type=get_enum_type(SUPPORTED_LANGUAGES), help='The language to be used to create the bot.', arg_group='Web Bot Specific')
-        c.argument('version', options_list=['-v', '--version'], help='The Microsoft Bot Builder SDK version to be used to create the bot', arg_type=get_enum_type(['v4']), arg_group='Web Bot Specific')
```

Comment:
Is it a breaking change? Be careful with it. Add it to history, and add BREAKING CHANGE.
You can also use deprecate argument.
https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#deprecating-commands-and-arguments

Reply:
This PR feedback regarding the history still needs to be added into #11318: command module is in preview. We only support v4 bots with the latest change, and use it by default so the flag `--version` is unnecessary.